samples/UDPEchoServ4.js

```diff
@@ -0,0 +1,14 @@
+var PORT = 33333;
```

Comment:
I guess it would help me, so we don't wrongly slap an Intel copyright on it at some point.